tasks/clamav-scan.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: clamav-scan
+spec:
+  params:
+    - name: image-digest
+      description: Image digest to scan
+    - name: image-url
+      description: Url to image
+  steps:
+    - name: extract-and-scan-image
+      image: quay.io/redhat-appstudio/hacbs-test:v1.0.1
+      script: |
+        # copy virus database from sidecar
+        cp -r /tmp/clamdb/* /var/lib/clamav
+
+        # strip new-line escape symbol from parameter and save it to variable
+        imageanddigest=$(echo '$(params.image-url)'@'$(params.image-digest)' | tr -d '\n')
+
+        mkdir -p /tmp/extracted-image && cd /tmp/extracted-image
+        oc image extract -a /workspace/registry-auth/.dockerconfigjson $imageanddigest
+        clamscan /tmp/extracted-image -ri --max-scansize=250M | tee /tekton/home/clamscan-result.log
+        echo "Executed-on: Scan was executed on version - $(clamscan --version)" | tee -a /tekton/home/clamscan-result.log
+      volumeMounts:
+        - mountPath: /tmp/clamdb
+          name: dbfolder
+    - name: modify-clam-output-to-json
+      image: python
```

Comment:
do not use generic image from dockerhub.

Reply:
Enhancing hacbs-test image with python binaries at the moment. Will update image reference once it get merged.